### ISSUE TITLE
PrimeNg - Improve accessibility by setting inputId

### ISSUE THREAD

**Issue by srollinet** [ngx-formly/ngx-formly#3899]
**Is your feature request related to a problem? Please describe.**

When testing the accessibility on a formly form using PrimeNg components, a lot of issues related to missing labels are found.
This is because the `inputId` prop is not set in the templates.

By setting this prop, the labels are correctly attached to the controld, using `for=id`

**Describe the solution you'd like**
The templates should be modified to set `[inputId]="id"` when applicable

**Describe alternatives you've considered**
-

**Additional context**
-

I can make a PR if you want

**Comment by aitboudad** [comment]
> I can make a PR if you want

Yes please
